src/core/BufferAttribute.js

```diff
@@ -310,6 +311,14 @@ BufferAttribute.prototype = {
 
 		return new this.constructor().copy( this );
 
+	},
+
+	discard: function () {
+
+		var oldArray = this.array;
+
+		this.array = new oldArray.constructor( 1 ); // create dummy minimal length TypedArray
```

Comment:
I'm trying to avoid changes to the WebGLRenderer, which uses instanceof buffer.array and buffer.array.BYTES_PER_ELEMENT in setupVertexAttributes().  bytesPerElement could be made a property of the attribute, but I'm not sure what the best way to handle the instanceof case, apart from an intermediate type value which seems equally clumsy.

Your suggestions?